src/components/MapContainer/MapContainer.js

```diff
@@ -1,123 +1,90 @@
-import React, { Component } from "react";
+import React from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
 import { connect } from "react-redux";
-import { compose } from "redux";
-import { GoogleApiWrapper, InfoWindow } from "google-maps-react";
+import { GoogleLogout } from "react-google-login";
 
 import { marksFetchData } from "../../actions/marksActions";
 import { userLogout } from "../../actions/signInActions";
-import { GoogleLogout } from "react-google-login";
 
 import "./MapContainer.scss";
 
-import CurrentLocation from "../CurrentLocation/CurrentLocation";
+import MapBoxContainer from "../MapBoxContainer/MapBoxContainer";
 import RestaurantDetails from "../RestaurantComponents/RestaurantDetails";
 
-export class MapContainer extends Component {
-  // componentWillMount() {
-  //   // let data = {
-  //   //   userId: '5c7015b00b10a5189ccc07e2',
-  //   //   groupId: '5c7016010b10a5189ccc07e3'
-  //   // }
+class MapContainer extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    // get user last joined group
+    this.groupId = this.props.getUserData.userGroups[
+      this.props.getUserData.userGroups.length - 1
+    ];
+  }
 
-  //   // this.props.getUserData(data)
-  //   this.props.marksFetchData("https://map-share-dev-api.herokuapp.com/api/marks?");
-  // }
-  groupId = this.props.getUserData.userGroups[
-    this.props.getUserData.userGroups.length - 1
-  ];
   componentDidMount() {
-    console.log("fetch first time");
+    //fetch user's group id marks
     this.props.marksFetchData(
       "https://map-share-dev-api.herokuapp.com/api/marks?groupId=" +
```

Comment:
the backend is deployed on heroku which takes time to load. Also, you have to run locally on port 3000 since thats the google login permission I set. I think I should probably mention that in the README file